terraform/application/config/production.tfvars.json

```diff
@@ -0,0 +1,11 @@
+{
+    "cluster": "production",
+    "namespace": "tra-production",
+    "evidence_container_retention_in_days": 30,
+    "enable_monitoring": true,
+    "external_url": "https://access-your-teaching-qualifications.education.gov.uk/healthcheck",
+    "statuscake_contact_groups": [282453],
+    "replicas": 2,
+    "azure_enable_high_availability": true,
+    "azure_enable_monitoring": true
+}
```

Comment:
Can we use the same DB type as the existing? See https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/terraform/workspace_variables/production.tfvars.json#L12

Reply:
https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/terraform/application/variables.tf#L33C11-L33C41 should be set to true so we can upload DB dumps

Reply:
Could you please `"send_traffic_to_maintenance_page": false`. It's useful to have it here in case of a DR scenario

Reply:
Can we set account_replication_type to GRS. It's safer for prod